docs/products/metrics/concepts/storage-resource-scaling.md

```diff
@@ -21,21 +28,54 @@ that rely on disk storage:
 
 - **Thanos Metric Receiver**: This component is the initial point of contact for your
   metrics. It accepts incoming data streams, temporarily stores them in a
-  local cache using disk space, and transfers them to object storage at
+  local cache using disk space, and transfers them to tiered storage at
   regular intervals.
 - **Thanos Metric Store**: The store component tracks the location of your metric
-  data shards within the object storage. It also maintains a small amount of
+  data shards within the tiered storage. It also maintains a small amount of
   information about these remote blocks on the local disk, ensuring it stays
-  synchronized with the object storage bucket.
+  synchronized with the tiered storage.
 - **Thanos Metric Compactor**: The compactor component is crucial in optimizing
   long-term data storage usage. It periodically downloads data chunks from object
   storage, performs downsampling (reducing data granularity for older data), and
-  uploads the compacted data back to object storage. This process requires temporary
+  uploads the compacted data back to tiered storage. This process requires temporary
   storage of the downloaded data on the local disk.
 
 The amount of disk space needed varies depending on your data volume and complexity.
 Aiven's service plans are designed to handle the most typical use cases.
 
+## Storage costs and billing
+
+Aiven for Metrics storage costs consist of two components:
+
+- **Local disk storage**:  Included in the base service plan to temporarily store or
+  process historical data and caching.
+- **Data stored in tiered storage**: Billed based on the highest amount of data retained
+  in tiered storage during each billing period.
+
+**Example billing scenario**
+
+If you use a **Start-16** plan with **640 GB** of local disk storage, your
+billing includes:
+
+- The base cost of the Start-16 plan.
+- Additional tiered storage costs, determined by the volume of metrics data retained.
+
+The local disk is used to temporarily store or process historical data to improve
```

Comment:
nit: This looks a little out of place in the example section.

Reply:
Thanks! I’ve removed this since it repeats what is already mentioned in the line above.